Binary files wxorig/.DS_Store and wxnew/.DS_Store differ
Binary files wxorig/src/.DS_Store and wxnew/src/.DS_Store differ
Binary files wxorig/src/common/.DS_Store and wxnew/src/common/.DS_Store differ
diff -uNr wxorig/src/common/utilscmn.cpp wxnew/src/common/utilscmn.cpp
--- wxorig/src/common/utilscmn.cpp	2010-11-05 08:58:46.000000000 -0400
+++ wxnew/src/common/utilscmn.cpp	2010-12-30 16:26:34.000000000 -0500
@@ -115,6 +115,8 @@
 // Convert 2-digit hex number to decimal
 int wxHexToDec(const wxString& str)
 {
+    if (str.Length() < 2)
+        { return 0; }
     char buf[2];
     buf[0] = str.GetChar(0);
     buf[1] = str.GetChar(1);
